components/browser/engine-gecko-beta/src/test/java/mozilla/components/browser/engine/gecko/GeckoEngineSessionTest.kt

```diff
@@ -1267,6 +1267,7 @@ class GeckoEngineSessionTest {
             override fun onLoadRequest(
                 engineSession: EngineSession,
                 uri: String,
+                lastOnLoadRequestUri: String?,
```

Comment:
We should test that we pass the right value (previous lost request uri).

Reply:
Agreed.  I'll add the test.